src_c/display.c

```diff
@@ -2096,7 +2100,7 @@ pg_toggle_fullscreen(PyObject *self, PyObject *_null)
     if (!win)
         return RAISE(pgExc_SDLError, "No open window");
 
-    flags = SDL_GetWindowFlags(win) & SDL_WINDOW_FULLSCREEN_DESKTOP;
+    flags = SDL_GetWindowFlags(win);
     /* SDL_WINDOW_FULLSCREEN_DESKTOP includes SDL_WINDOW_FULLSCREEN */
```

Comment:
Now that the line above is changed, the comment just below it seems a bit out of place. Should it be moved elsewhere?